packages/server/src/app.js

```diff
@@ -10,6 +10,7 @@ import websiteMiddleware from './middleware/website';
 import graphiqlMiddleware from './middleware/graphiql';
 import graphqlMiddleware from './middleware/graphql';
 import errorMiddleware from './middleware/error';
+import i18Init from '../src/i18n/index';
```

Comment:
There is no such thing as `i18`, there is `i18n`, please rename this variable accordingly

Reply:
Fixed.